packages/verified-fetch/src/types.ts

```diff
@@ -1 +1,3 @@
 export type RequestFormatShorthand = 'raw' | 'car' | 'tar' | 'ipns-record' | 'dag-json' | 'dag-cbor' | 'json' | 'cbor'
+
+export type SupportedBodyTypes = string | ArrayBuffer | Blob | ReadableStream<Uint8Array> | null
```

Comment:
Picking nits but `Types` is redundant in a type, `Supported` is a business-logic kind of decision not a type, so just `Body`?

```suggestion
export type Body = string | ArrayBuffer | Blob | ReadableStream<Uint8Array> | null
```

Reply:
I would rather do something like `ResponseBody`, but i'm good with any.

`Body` could easily be a type that comes from builtin/global types that could cause confusion. `SupportedBody` would be better I guess.. but it's explicitly informing devs reading the code that it's not just typical `Response.body` types.

Reply:
`ResponseBody` would be fine.  It's a minor point tbh.